bin/main.rb

```diff
@@ -0,0 +1,175 @@
+require_relative '../lib/player'
+
+require_relative '../lib/game'
+
+game = Game.new
+
+board = game.game_arr
+
+def welcome_screen(game, board)
+  name = ''
+
+  puts 'Welcome to TIC-TAC-TOE'
+
+  puts "First player, What's your name?"
+
+  loop do
+    name = gets.chomp
+
+    unless game.name_check(name)
+
+      puts "Your name cannot be an empty line or a number \n \n \n"
+
+      puts "Please enter a valid name \n"
+
+    end
+
+    break if game.name_check(name) == true
+  end
+
+  game.player1.name = name
+
+  puts "Hi #{game.player1.name} , your key is #{game.player1.key} "
+
+  puts "Second player, What's your name?"
+
+  loop do
+    name = gets.chomp
+
+    unless game.name_check(name)
+
+      puts "Your name cannot be an empty line or a number \n \n"
+
+      puts "Please enter a valid name \n"
+
+    end
+
+    break if game.name_check(name) == true
+  end
+
+  game.player2.name = name
+
+  puts "Hi #{game.player2.name} , your key is #{game.player2.key} "
+
+  display_board(board)
```

Comment:
- [ ] You can do like this as well:

![image](https://user-images.githubusercontent.com/61479648/94426371-dad95b80-0163-11eb-9b14-badc2d374871.png)